bin/start

```diff
@@ -29,6 +29,10 @@ function filterUp() {
     grep -E -v 'is up-to|--remove-orphans'
 }
 
+function testWebsiteConnection() {
+	printf "GET / HTTP/1.1\r\nHost: localhost:3020\r\n\r\n" | nc -w 1 localhost 3020 | grep -c '<body>' || true
```

Comment:
IIRC the `3020` is configurable (via an ENV var maybe? Dunno), so maybe we need to use that config here too?

Reply:
It is configurable, but for that you'd have to edit the port mapping in the `docker-compose-full.yml` file. There is a Docker CLI command to retrieve the port, but that requires the container to already be running, which we could do here, but there are other mentions of the website port in the text that is echoed. I could use some regex magic to extract the data from the compose file, but that is quite ugly.

Reply:
I'll merge this and then we can decide what to do with this later. I think we could configure the port as an environment variable if we want.

Reply:
> It is configurable, but for that you'd have to edit the port mapping in the docker-compose-full.yml file.

No, you'd just edit `stack.yml` and let it update the port mappings.  So you could maybe check `stack.yml` but now you'r switching from Bash to Ruby to parse the yams?  Anyone using a custom port should really be doing it by updating stack.yml, not `compose-full`.

Reply:
@ErikSchierboom Seems like it's probably best to open an issue for this that someone can tackle when they want to change the port and use these scripts?

Reply:
@iHiD https://github.com/exercism/development-environment/issues/27